### PR TITLE
hw-manager: add support for Nicole-x86

### PR DESCRIPTION
**Pull request by shamanfree**
Add information and fix regexp for Nicole-x86

Signed-off-by: Georgii Matyukhin <g.matyukhin@yadro.com>

### CODE REVIEW

File: src/hw/product_registry.hpp [hunk 1]
```diff
@@ -454,4 +464,8 @@ static const std::vector<ProductDescription> productRegistry = {
      .productName = "TATLIN.ARCHIVE.S",
      .fans = FanConnectorsS320,
      .sysFanPN = "ASMFAN781103A",
-     .cpuFanPN = "CPUHSN792004A"}};
+     .cpuFanPN = "CPUHSN792004A"},
+    {.pnameRegex = std::regex("TATLIN UNIFIED GEN2.*"),
```
Comment (nest1ing) [inline src/hw/product_registry.hpp]: I dislike this value, but it should be discussed with PM-s.

**Review by nest1ing — DISMISSED** [review]
Looks good to me except the product name.

File: src/hw/product_registry.hpp [hunk 2]
```diff
@@ -454,4 +464,8 @@ static const std::vector<ProductDescription> productRegistry = {
      .productName = "TATLIN.ARCHIVE.S",
      .fans = FanConnectorsS320,
      .sysFanPN = "ASMFAN781103A",
-     .cpuFanPN = "CPUHSN792004A"}};
+     .cpuFanPN = "CPUHSN792004A"},
+    {.pnameRegex = std::regex("NICOLE-X86.*"),
+     .productName = "NICOLE-X86",
+     .fans = FanConnectorsNicoleX86,
+     .sysFanPN = "NODE_CPU_FAN"}};
```
Comment (AlexanderAmelkin) [inline src/hw/product_registry.hpp]: Why a system fan PN is set to 'CPU Fan' ? Isn't that what `.cpuFanPN` is for?

Comment (shamanfree) [reply src/hw/product_registry.hpp]: Based on the results of the discussion with Pavel, it became clear that it is supposed to cool the CPU by airflow from system fans, whose PN is unknown yet. But here is a typo made by me, it should be "NODE_SYS_FAN". Is it OK for now?

**Review by nest1ing — APPROVED** [review]

**Review by AlexanderAmelkin — APPROVED** [review]

**Pull request merged**